WordPress/Classes/ViewRelated/Media/MediaLibraryViewController.swift

```diff
@@ -255,6 +255,11 @@ class MediaLibraryViewController: WPMediaPickerViewController {
     private func showOptionsMenu() {
         let menuAlert = UIAlertController(title: nil, message: nil, preferredStyle: UIAlertControllerStyle.actionSheet)
 
+        if blog.isQuotaAvailable, let quotaPercentageUsedDescription = blog.quotaPercentageUsedDescription, let quotaSpaceAllowedDescription = blog.quotaSpaceAllowedDescription {
+            let formatString = NSLocalizedString("%@ of %@ used.", comment: "Amount of disk quota being used. First argument is the total percentage being used second argument is total quota allowed in GB.Ex: 33% of 14 GB used.")
```

Comment:
A bit of a nitpick, but could we remove the period from the end of the string (so it's just `%@ of %@ used`), so it matches Calypso / the designs?